kernel/armv7a/kernel_sgemm_4x4_lib4.S

```diff
@@ -1178,8 +1199,8 @@ _kernel_sgemm_nt_4x4_lib4:
 	ldr		r5, [fp, #0] // beta
 	ldr		r6, [fp, #4] // C
 
-#if MACRO_LEVEL>=1
-	INNER_SCALE_AB_4X4_LIB4
+#if MACRO_LEVEL>=2
+	INNER_SCALE_AB_4X4_LIB4 .LC00_kernel_sgemm_nt_4x4_lib4
```

Comment:
I'm not familiar with macro parameters, but the ones you used above, the parameter was passed as
`INNER_SCALE_AB_4X4_LIB4(.LC00_kernel_sgemm_nt_4x4_lib4)`
Are they equivalent? Should this be fixed?

Reply:
I am assuming you are referring to the additional parentheses. I *think* they are fine, but they probably got inserted by accident because I cannot find documentation that this is actually supported syntax. In any case the one without parentheses seems to be the norm and for consistency reasons alone I'll remove all parentheses in question.